### PR TITLE
Add image_size and project_name interfaces in blueoil convert

### PR DESCRIPTION
**Pull request by kchygoe**
## Motivation and Context
I want to export and convert models by blueoil easily.

## Description
Added options in blueoil/cli.py and blueoil/blueoil_convert.py
Because both files have main interfaces, I added options in both files.

## How has this been tested?

Tested with `make test`

and followings
blueoil cli
```
Usage: blueoil convert [OPTIONS]
Convert trained model to binary files.

Options:
-e, --experiment_id TEXT        ID of this experiment.  [required]
-p, --checkpoint TEXT           Checkpoint name. e.g. save.ckpt-10001
[required]
-t, --template TEXT             Path of output template directory.
--image_size <INTEGER INTEGER>...
input image size height and width. if these
are not provided, it restores from saved
experiment config.e.g --image_size 320 320
--project_name TEXT             project name which generated by convert
--help                          Show this message and exit.
```

blueoil_convert.py
```
Usage: blueoil_convert.py [OPTIONS]
Options:
-i, --experiment_id TEXT        id of this experiment.  [required]
--restore_path TEXT             restore ckpt file base path. e.g.
saved/experiment/checkpoints/save.ckpt-10001
--image_size <INTEGER INTEGER>...
input image size height and width. if these
are not provided, it restores from saved
experiment config.e.g --image_size 320 320
--project_name TEXT             project name which generated by convert
-h, --help                      Show this message and exit.
```

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature / Optimization (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: blueoil/blueoil_convert.py [hunk 1]
```diff
@@ -115,20 +119,20 @@ def run(experiment_id, restore_path, output_template_dir=None):
...
 
     # Export model
-    export_dir = run_export(experiment_id, restore_path=restore_path)
+    export_dir = run_export(experiment_id, restore_path=restore_path, image_size=image_size)
 
     # Set arguments
     input_pb_path = os.path.join(export_dir, "minimal_graph_with_shape.pb")
-    dest_dir_path = export_dir
-    project_name = "project"
+    if project_name is None or len(project_name) < 1:
```
Comment (iizukak) [inline blueoil/blueoil_convert.py]: Do you want to check empty string?

```
if not project_name:
```

is simple.

```
In [6]: not None                                                                                                                                                                                                                            
Out[6]: True

In [7]: not ""                                                                                                                                                                                                                              
Out[7]: True

In [8]: not "Hoge"                                                                                                                                                                                                                          
Out[8]: False
```

Comment (kchygoe) [reply blueoil/blueoil_convert.py]: THX!

Comment (kchygoe) [reply blueoil/blueoil_convert.py]: fixe! thank you for the comment :)

**Comment by iizukak** [comment]
@tkng We need readability reviewer.

**Comment by tkng** [comment]
- [x] readability approval

**Pull request merged**